lib/ultrajson.h

```diff
@@ -259,6 +259,7 @@ typedef struct __JSONObjectEncoder
   /*
   Configuration for spaces of indent */
   int indent;
+  const char* indent_chars;
```

Comment:
This should be `indentChars`, as all member names use `camelCase` not `snake_case`.